### PR TITLE
PSE-7615: Fixing theme for libsass upgrade

### PR DESCRIPTION
**Pull request by JaminsonR**
### Description

This PR updates `sass-loader` to `^8.0.2` and `sass` to `^1.77.1` so this theme is compatible with the bentobox dependency `libsass 0.23.0`

[QA Doc](https://docs.google.com/document/d/1YEE78VTAHemq862WIw6PVc3tZP5cDOhIhs8dsuy5cic/edit)

### Ticket Link

[PSE-7615](https://getbentobox.atlassian.net/browse/PSE-7615)
Bentobox PR for test in dockerized staging with  `libsass 0.23.0`: https://github.com/getbento/bentobox/pull/19363

### Checklist:

- [x] This PR has been self-reviewed (for code styling, misspellings, console logs, commented out code, etc.) before assigning reviewers.
- [x] This PR fulfills all the acceptance criteria.
- [ ] Documentation has been created/updated, as needed.
- [ ] Tests have been added/updated/deleted.
- [x] Sensitive data is not logged in plain text.

#### Tips & Notes

- Draft mode will prevent tests from running.  Consider opening/keeping your PR in draft mode until your code is ready for testing to reduce CI congestion.

- If your PR contains migrations update your branch with the latest changes in `master` and run tests one last time before merging it.  This will help reduce migration conflicts on `master` if someone merges a PR that also has migrations.

[PSE-7615]: https://getbentobox.atlassian.net/browse/PSE-7615?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -77,13 +76,10 @@ module.exports = {
...
       // BrowserSync options
       {
-        // browse to http://localhost:3000/ during development
-        host: 'localhost',
-        port: 3000,
         // proxy the Webpack Dev Server endpoint
         // (which should be serving on http://localhost:3100/)
         // through BrowserSync
-        proxy: 'http://127.0.0.1:5000',
+        proxy: 'https://littlespain.localtest.me:8000/',
```
Comment (Dario0117) [inline webpack.config.js:82]: Should this be specific for this single account?

**Comment by Dario0117** [comment]
The new dependency versions are backwards compatible with the current version on bentobox master?

File: webpack.config.js (hunk 1 above)
Comment (JaminsonR) [reply webpack.config.js:82]: this is the only site in the database that currently uses this theme, so no other URL will show your changes to the theme

**Comment by JaminsonR** [comment]
> The new dependency versions are backwards compatible with the current version on bentobox master?

@Dario0117 that is correct, I have tested this on my latest change to the dockerized stating env here: https://littlespain.pse-7615fixing-libsass.getbento.dev/

**Review by Dario0117 — APPROVED** [review]

File: assets/dist/assets.json [hunk 2]
```diff
@@ -1 +1 @@
-{"main":{"css":"/assets/dist/style.css","js":"/assets/dist/main.js"}}
\ No newline at end of file
+{"main":{"css":"/assets/dist/style-af1035.css","js":"/assets/dist/main.js"}}
```
Comment (jguilindro) [inline assets/dist/assets.json:1]: Does this have to be updated with the new SHA everytime we make changes?

**Review by wllamasr — APPROVED** [review]

**Review by jguilindro — APPROVED** [review]

File: assets/dist/assets.json (hunk 2 above)
Comment (JaminsonR) [reply assets/dist/assets.json:1]: These changes come from running `npm run build` so I think so

**Pull request merged**